js/views/userPage/Home.js

```diff
@@ -19,33 +33,86 @@ export default class extends BaseVw {
   events() {
     return {
       'click .js-termsLink': 'termsClick',
-      'click .js-termsClose': 'termsClose',
-      'click .js-addModerator': 'addModeratorClick',
+      'click .js-addModerator': 'changeModeratorClick',
       'click .js-guid': 'guidClick',
       'mouseleave .js-guid': 'guidLeave',
     };
   }
 
+  get ownMod() {
```

Comment:
Since we're repeating this, can we make an isOwnModerator(guid) method on the settings model?